recipe/meta.yaml

```diff
@@ -56,12 +61,15 @@ outputs:
   - name: {{ name }}
     build:
       script: python -m pip install .[empy,report-timings] --no-deps --ignore-installed -vv
```

Comment:
```suggestion
      script: {{ PYTHON }} -m pip install .[empy,report-timings] --no-deps --ignore-installed -vv
```

I think they prefer this now...

Reply:
For some reason that doesn't work in the `outputs` section.

Reply:
(changed here - https://github.com/conda-forge/cylc-flow-feedstock/pull/29/files)

Reply:
Okay, pls ignore :)